apps/test-suite/components/SuiteResult.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import SpecResult from './SpecResult';
+
+export default class SuiteResult extends React.Component {
```

Comment:
Since you use function components elsewhere, should be straightforward to do the same here.